css/style.css

```diff
@@ -5,3 +5,22 @@ html {
 }
 
 /*put all other styles below this line, please*/
+#body {
+  display:grid;
+}
+
+#navigation {
+  position: sticky;
+}
+
+#header {
+  width: min-content;
```

Comment:
`min-content` _et al_ should be set up in the `body` selector's CSS... they're going to be a part of the top-level `grid`'s `grid-template-rows`